filter_mitm_logs.py

```
@@ -0,0 +1,115 @@
+        # Remove trailing commas
+        line = line.replace(",}", "}")
+        line = line.replace(",]", "]")
+
+        # Fix incomplete strings
+        line = line.replace('":,', '":null,')  # Handle cases where value is missing after a key
+    except Exception as e:
+        print(f"Error during cleaning: {e}")
+    return line
+
+
+def filter_jsonl_file(input_file, output_file, url):
```

Ruff (0.8.2)

`filter_jsonl_file` is too complex (15 > 10)

(C901)